src/components/Restoration/RestorationType.vue

```diff
@@ -86,6 +86,10 @@
           </v-radio-group>
         </v-col>
       </v-row>
+
+      <!-- Vuetify Divider Line b/w restoration and approval type -->
+      <v-divider></v-divider>
```

Comment:
So this puts a line at the bottom of the restoration type component, but what if approval type is not the next component? Then you'd get a line you don't want.

It would be better to put this line in the parent component, where it's clear there is the restoration type component followed by the approval type component and you want something in between. The principle here is that the parent should be responsible for layout of its components (including separators).

The v-divider itself looks good to me (from your screenshots).

Reply:
Looks OK in temp URL other than you might want some left/right padding/margin on the divider so it doesn't go edge-to-edge (should go to existing margins).

![image](https://github.com/bcgov/business-create-ui/assets/10002889/8370d84a-acf5-4618-97e7-1fe362432186)